README.md

```diff
@@ -190,12 +190,12 @@ This UI consumes, and is therefore dependent on, data from the following APIs:
-| UPDATE_INDUCTION_SKILLS_AND_INTERESTS_SECTION_ENABLED          | false         | Boolean | Set to true to enable update of Skills & Interests section of Induction via PLP screens.                                                                                          |
-| UPDATE_INDUCTION_WORK_EXPERIENCE_SECTION_ENABLED               | false         | Boolean | Set to true to enable update of Work Experience section of Induction via PLP screens.                                                                                             |
-| UPDATE_INDUCTION_WORK_INTERESTS_SECTION_ENABLED                | false         | Boolean | Set to true to enable update of Work Interests section of Induction via PLP screens.                                                                                              |
-| UPDATE_INDUCTION_TRAINING_INTERESTS_IN_PRISON_SECTION_ENABLED  | false         | Boolean | Set to true to enable update of Training Interests In Prison section of Induction via PLP screens.                                                                                |
-| NEW_COURSE_AND_QUALIFICATION_HISTORY_ENABLED                         | false         | Boolean | Set to true to show the new In-prison qualifications and achievements history on the Education & training tab.                                                                                               |
+| Name                                                  | Default Value | Type    | Description                                                                                                    |
+|-------------------------------------------------------|---------------|---------|----------------------------------------------------------------------------------------------------------------|
+| SOME_TOGGLE_ENABLED                                   | false         | Boolean | Example feature toggle, for demonstration purposes.                                                            |
+| INCLUDE_PRISON_TIMELINE_EVENTS_ENABLED                | false         | Boolean | Set to true to include Prison Movement related events within a Prisoner's Timeline.                            |
+| UPDATE_INDUCTION_SKILLS_AND_INTERESTS_SECTION_ENABLED | false         | Boolean | Set to true to enable update of Skills & Interests section of Induction via PLP screens.                       |
+| UPDATE_INDUCTION_WORK_EXPERIENCE_SECTION_ENABLED      | false         | Boolean | Set to true to enable update of Work Experience section of Induction via PLP screens.                          |
+| UPDATE_INDUCTION_WORK_INTERESTS_SECTION_ENABLED       | false         | Boolean | Set to true to enable update of Work Interests section of Induction via PLP screens.                           |
+| UPDATE_INDUCTION_IN_PRISON_TRAINING_SECTION_ENABLED   | false         | Boolean | Set to true to enable update of In-prison Education & Training Interests section of Induction via PLP screens. |
```

Comment:
This might seem minor, but we have so many "interests" and "work" related areas of an induction (e.g. pre and post prison) that I wanted `IN_PRISON` to appear at the beginning of the name. Ideally this would be `UPDATE_INDUCTION_IN_PRISON_EDUCATION_AND_TRAINING_SECTION_ENABLED`, but it's a bit long. I've therefore decided to settle on `TRAINING`, as that's what the page is really about - i.e. "what 'training' would the prisoner like to do in prison?"